docs/webpack.md

```diff
@@ -0,0 +1,15 @@
+# Entry Points and Globally Exposing Objects
+
+In `webpack.server.rails.config.js`, you should ensure you config the entry points correctly.
+
+When `React.version >= '0.14'`, the entry points might include below:
```

Comment:
@robwise Maybe you can reword this a bit.

When using React 0.14 and greater you **need** both include `react-dom/server` and `react` as values for `entry`, like this:

```js
  entry: {

    // See use of 'vendor' in the CommonsChunkPlugin inclusion below.
    vendor: [
      'babel-core/polyfill',
      'jquery',
      'jquery-ujs',
      'react',
      'react-dom',
    ],
```

and you need to expose them:

```js
      // React is necessary for the client rendering:
      {test: require.resolve('react'), loader: 'expose?React'},
      {test: require.resolve('react-dom'), loader: 'expose?ReactDOM'},
      {test: require.resolve('jquery'), loader: 'expose?jQuery'},
      {test: require.resolve('jquery'), loader: 'expose?$'},
```

webpack.server.config.js is similar, but substitute 

```js
 entry: ['./yourCode', 'react-dom/server', 'react'],
```
and use this line rather than 'react-dom':

```js
   {test: require.resolve('react-dom/server'), loader: 'expose?ReactDOMServer'},
```

## version 0.13

You don't need to put in `react-dom`.